src/style/themeManager.ts

```diff
@@ -38,6 +41,14 @@ export class ThemeManager {
     return _.get(this.theme, key);
   }
 
+  setThemeContext(context: Context<any>) {
```

Comment:
Where will you set the theme context? in the private config files?

Reply:
Indeed. 
If you don't set, nothing happens.. 
This is basically a good API for us and other consumer to have global theme context